tests/main/desktop-portal-filechooser/task.yaml

```diff
@@ -59,7 +59,13 @@ execute: |
     touch /tmp/file-to-write.txt
     chown test:test /tmp/file-to-write.txt
     as_user test-snapd-portal-client save-file "from-sandbox"
```

Comment:
Since this is not running in the background I think it is highly fishy that a write can finish without being visible here.

CC @jhenstridge for ideas.

Reply:
This is surprising to me.  Looking at the underlying code, a close of the fuse file descriptor should also close the underlying file.  Is it just one particular distro release we're seeing the failure on?

Reply:
I don't have the data on hand to answer. I will record the failures as I see them starting from now,.

Reply:
@jhenstridge @zyga 
These are the last failures during the last 2 weeks for this test and all of them are happening on ubuntu-18.04-64:
https://travis-ci.org/snapcore/snapd/jobs/578664831#L3446
https://travis-ci.org/snapcore/snapd/jobs/578176116#L6804
https://travis-ci.org/snapcore/snapd/jobs/576845090#L1288
https://travis-ci.org/snapcore/snapd/jobs/575899118#L6118

Reply:
2 errors today always on ubuntu-18.04-64:
https://travis-ci.org/snapcore/snapd/jobs/580112858#L8998
https://travis-ci.org/snapcore/snapd/jobs/580073815#L6135